manager-bundle/src/HttpKernel/ContaoCache.php

```diff
@@ -21,23 +21,50 @@
 use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Terminal42\HeaderReplay\SymfonyCache\HeaderReplaySubscriber;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
 use Toflar\Psr6HttpCacheStore\Psr6Store;
 
 class ContaoCache extends HttpCache implements CacheInvalidation
 {
     use EventDispatchingHttpCache;
 
+    /**
+     * @var bool
+     */
+    private $cacheWasCircumvented;
+
     public function __construct(ContaoKernel $kernel, string $cacheDir = null)
     {
         parent::__construct($kernel, $cacheDir);
 
         $this->addSubscriber(new PurgeListener());
         $this->addSubscriber(new PurgeTagsListener());
-        $this->addSubscriber(new HeaderReplaySubscriber(['ignore_cookies' => ['/^csrf_./']]));
         $this->addSubscriber(new CleanupCacheTagsListener());
     }
 
+    public function wasCacheCircumvented(): bool
```

Comment:
You might wonder why I did not mock the kernel and check if `->handle()` was called or not. I did do this initially but it doesn't really work. I only want to check if our `ContaoCache` does what it's supposed to do. If a private request comes in `->kernel->handle()` should not be called, thus you would set `->expects($this->never())` but that will actually not work because the request will flow through the `HttpCache` that has no cache entry eventually being `forward()`ed again the the kernel so that expectation would fail 😄